vhdbuilder/packer/components.json

```diff
@@ -230,13 +218,9 @@
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.26.19",
         "v1.26.22",
```

Comment:
could also remove v1.26.22 since AKS v1.26 is not supported any more

Reply:
Removed.